client/me/help/help-courses/index.jsx

```diff
@@ -58,14 +60,21 @@ function getCourses() {
 	];
 }
 
+function mapDispatchToProps( dispatch ) {
+	// This function only adds a way of dispatching courses because we don't have another mechanism yet.
+	// Once the courses make it into the API this function should go away in preference for
+	// something linke <QueryHelpCourses />
+	return {
+		fetchCourses: () => dispatch( receiveHelpCourses( getCourses() ) )
```

Comment:
My next PR will be to move this into the API so it wont be here for long. Also I can't load it as the default state because it needs to be translated. This also points out that right now if the user changes their language the text here won't be updated as it should. I'll fix that before merging. :)